.github/workflows/env-files/update-lockfiles.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Execute this script to update all lock files to the latest versions of dependencies.
+
+rm *-conda-lock.yml
+
+for python_version in 3.9 3.10. 3.11 3.12
```

Comment:
```suggestion
for python_version in 3.9 3.10 3.11 3.12
```
Extra dot after 3.10